### DISCUSSION TITLE
How to make Twitter preview images

### DISCUSSION THREAD

**Discussion by MattDodsonEnglish** [thuliteio/doks#532] [Q&A]
First of all, thanks for the great work. We are proud of how our site looks.

Recently, my colleague tried to share a blog post he wrote on Twitter. However, no image shared in the Twitter preview. 

Finally I seem to have gotten the image preview to work, but I did it in complete "chaos monkey" fashion,
by hardcoding a link to the image in the params and in the twittercard partial:

- https://github.com/TrueBlocks/trueblocks-docs/blob/master/config/_default/params.toml
- https://github.com/TrueBlocks/trueblocks-docs/blob/master/layouts/partials/head/twittercard.html

But, I don't think this is the best way. Besides, I'd also like to figure out how to do previews for individual pages, as I saw on the @getdoks Twitter account.

My questions are three:
- where do I configure the default twitter preview image?
- Is there a specific place in the repo where I should upload the default twitter image?
- How can we configure those nice, per-page twitter previews, to be cool like  @getdocs ?

For example, I'd like to do this: https://twitter.com/getdoks/status/1378062688887173122

I see that the preview image is hosted somewhere else, but I couldn't find what is pointing to the image in the GitHub repo:
https://github.com/h-enk/getdoks.org/blob/master/content/blog/doks-v02/index.md

**Comment by h-enk** [comment]
> How can we configure those nice, per-page twitter previews, to be cool like @getdocs ?

See: https://github.com/h-enk/doks/discussions/548
